packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -215,6 +226,16 @@ const styles: ComponentStyles<SiteManagedClasses, DevSiteDesignSystem> = {
     site_paneForm: {
         padding: toPx(12),
     },
+    site_paneNavigationCloseButton: {
```

Comment:
does this ever receive focus?

Reply:
it can, I didn't style it though but I could do the standard brand color outline.